### ISSUE TITLE
example build script doesn't do much building

### ISSUE THREAD

**Issue by brettimus** [uncletammy/doc-templater#6]
Neither the `beforeConvert` or `afterConvert` hooks are firing. All I get is an empty array of metadata printed to the console.

Just trying to build the sails docs locally! Any help is greatly appreciated. Know you're busy.

**Comment by prijindal** [comment]
I am also getting the same problem while trying to build sails docs

**Comment by brettimus** [comment]
@prijindal Check it out! Here's how you can build sails docs locally. 

https://github.com/balderdashy/sails-docs/issues/523#issuecomment-136823015

**Comment by mikermcneil** [comment]
Yeah sorry guys, my bad -:shame:- some of tests/docs are quite out of date in this repo due to my various meanderings and other abuses to this lovechild of @uncletammy and myself lovechild.  If y'all see anything in particular that's out of date, let me know and I'll fix it.  Thanks!

**Comment by uncletammy** [comment]
![graveyardcat](https://cloud.githubusercontent.com/assets/5298142/17083480/2d5f608a-519a-11e6-979e-71ba9f8df7a5.jpg)

**Comment by LimitlessGreen** [comment]
Sadly the issues of sails-docs are gone.
I'm interested at the build script too.

**Comment by mikermcneil** [comment]
> Sadly the issues of sails-docs are gone.

they moved into the main Sails repo (we consolidated issues to have a single queue).  That's allowed the Sails project to keep up with all new issue activity consistently at our weekly roundup (instead of closing, a green label system is in use) cc @eashaw @rachaelshaw 

That only covers balderdashy/sails and repos that have issues closed, so this one fell through the cracks.

> I'm interested at the build script too.

@LimitlessGreen Modern example of using the internals of doc-templater (much simpler w/ async/await), combined with a lot of the previously unreleased stuff happening behind the scenes on sailsjs.com: https://github.com/fleetdm/fleet/pull/827/files  (work in progress, but will be finished in a week or so max)
